src/components/liquality/ConfirmLiqualityTransaction.vue

```diff
@@ -177,11 +173,18 @@ import LiqualityTxBuilder from '@/middleware/TxBuilder/LiqualityTxBuilder';
     AdvancedData,
   },
 })
+
 export default class ConfirmLiqualityTransaction extends Vue {
   txId = '';
 
   rawTx = '';
 
+  bitcoinPrice = 0;
+
+  fixedDecimals = 2;
+
+  allOutputs = [];
+
```

Comment:
I don't see these fields been used. Do you need them?

Reply:
no, deleted it